app/helpers/tax_helper.rb

```diff
@@ -12,12 +12,15 @@ def display_taxes(taxable, display_zero: true)
     end
   end
 
-  def display_line_items_taxes(line_item, display_zero: true)
+  def display_line_items_taxes(
+    line_item, display_zero: true, enterprise_fee_adjustments: EnterpriseFeeAdjustments.new
+  )
+    enterprise_fee_adjustments.adjustments = line_item.enterprise_fee_adjustments
```

Comment:
I don't see how this is any less coupled, we're still instantiating `EnterpriseFeeAdjustments` in the same class, just in a different (I think more convoluted) way.

I have to admit I didn't know the "dependency injection" concept though. I looked it up and can see how it could be achieved by passing an instantiated object, but I don't see that happening here. I'm not sure if it's necessary.